### PR TITLE
Enhancements for supporting LIS hotfix release

### PR DESCRIPTION
**Pull request by iamshital**

### CODE REVIEW

**Comment by johnsongeorge-w** [comment]
@DivyaJyothiKasam : Review the changes.

File: LISISO/RPMS70/install.sh [hunk 1]
```diff
@@ -5,14 +5,18 @@
...
-fi
+#List of errata kernel that needs special handling due to KABI changes
+file_errata_kernel_list="errata_kernel_list"
+[ -f $file_errata_kernel_list ] && ekernel_list=$(cat $file_errata_kernel_list | cut -d"=" -f2)
+
+for ekernel in $ekernel_list;do
+	if ! IsInstalledKernelOlderThanErrataKernel $ekernel;then
+		break
+	fi
+	update_number=$(expr $update_number + 1)
```
Comment (santoshx) [inline LISISO/RPMS70/install.sh]: expr is deprecated. You may consider rewriting this using $((..))

File: LISISO/RPMS70/install.sh [hunk 2]
```diff
@@ -5,14 +5,18 @@
...
+update_number=0
 source ${LIS_INSTALL_BASE_DIR}/commonfunctions.sh
 
-if ! IsInstalledKernelOlderThanErrataKernel $errata_kerver_1; then
-        checkrpms
-        installbuildrpm
-else
-	installbuildrpm 1
-fi
+#List of errata kernel that needs special handling due to KABI changes
```
Comment (santoshx) [inline LISISO/RPMS70/install.sh]: May be a space after #

File: LISISO/RPMS73/install.sh [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
-	installbuildrpm
-elif ! IsInstalledKernelOlderThanErrataKernel $errata_kerver_2; then
-	installbuildrpm 1
-else
-	installbuildrpm 2
-fi
+for ekernel in $ekernel_list;do
+	if ! IsInstalledKernelOlderThanErrataKernel $ekernel;then
+		break
+	fi
```
Comment (santoshx) [inline LISISO/RPMS73/install.sh]: Mix of tabs and spaces

**Review by santoshx — APPROVED** [review]
I see some minor spacing errors other than looks good.

**Comment by johnsongeorge-w** [comment]
Addressed all the review comments. Enhanced the install and upgrade script to use the commonfunction script for determining the update number (https://github.com/LIS/lis-rpm-build-pipeline/commit/ee458ac93b62bac44324221e56d5b394d24c804a). 
The same changes is pulled in and PR is updated.

**Review by santoshx — APPROVED** [review]

**Comment by johnsongeorge-w** [comment]
New PR #61 raised.

**Pull request closed without merging**